site/docs/v1/tech/oauth/tokens.adoc

```diff
@@ -103,6 +103,10 @@ https://tools.ietf.org/html/rfc7519#section-4.1.1[RFC 7519 Section 4.1.1].
 [field]#permissions# [type]#[Array<String>]#::
 The permission granted to the recipient Entity by the target Entity. This claim is only present if permissions are associated with the grant and any requested permissions are found in the grant.
 
+[field]#scope# [type]#[String]#::
+The scope of the Access token. See scope defined by
+https://datatracker.ietf.org/doc/html/rfc6749#section-3.3[RFC 6749 Section 3.3] and referenced in Client Credentials Grant https://datatracker.ietf.org/doc/html/rfc6749#section-4.4[RFC 6749 Section 4.4]
+
```

Comment:
You don't need this `+`.

Reply:
I think this is a goof on Github's part. I show just one space, similar to all other fields under the API class.